features/register_client.feature

```diff
@@ -9,3 +9,31 @@ Feature: Register a client
   Scenario: Register a client
     When I register using an activation key
     Then I should see this client in spacewalk
+
```

Comment:
@meaksh  when we register trad-client, when does it happen the refresh stuff? we can try to run it